src/_help/specification-support/openapi-support/overlays.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Overlays
+---
+
+- TOC
+{:toc}
+
+_Note: The Overlays Specification of OpenAPI is currently in beta. It is [sufficiently stable for experimentation](https://github.com/OAI/Overlay-Specification?tab=readme-ov-file#current-status), however please be advised that it may change depending on the community feedback._
```

Comment:
```suggestion
> Note: The Overlays Specification of OpenAPI is currently in
> beta. It is [sufficiently stable for
experimentation](https://github.com/OAI/Overlay-Specification?tab=readme-ov-file#current-status),
> however please be advised that it may change depending on the
> community feedback.
{: .info}
```